features/steps/test_starter/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/hooks.py

```diff
@@ -1,19 +1,5 @@
 """Project hooks."""
-from typing import Any, Dict, Optional
-
-from kedro.framework.hooks import hook_impl
-from kedro.io import DataCatalog
 
 
 class ProjectHooks:
-    @hook_impl
-    def register_catalog(
-        self,
-        catalog: Optional[Dict[str, Dict[str, Any]]],
-        credentials: Dict[str, Dict[str, Any]],
-        load_versions: Dict[str, str],
-        save_version: str,
-    ) -> DataCatalog:
-        return DataCatalog.from_config(
-            catalog, credentials, load_versions, save_version
-        )
+    pass
```

Comment:
I'm thinking, would it be better to delete the file altogether? The alternative is to leave it as example only (with commented out code), but that might be annoying for users to have a file that they'd have to delete?

Reply:
I considered it, but thought it would make more sense to do it as part of [dropping support for registration hooks](https://jira.quantumblack.com/browse/KED-2504)? I don't mind doing it here, but it would increase the scope of the PR.